### DISCUSSION TITLE
bypass age restricted

### DISCUSSION THREAD

**Discussion by pasindu20** [lavalink-devs/Lavalink#724] [Q&A] [answered]
how do i bypass age restricted or region not available tracks

```js
2022-08-25 20:03:28.722  WARN 12 --- [back-1-thread-1] c.s.d.l.s.y.YoutubeAccessTokenTracker    : YouTube auth tokens can't be retrieved because email and password is not set in YoutubeAudioSourceManager, age restricted videos will throw exceptions
``` 
also how do i fix this

**Comment by topi314** [comment, marked as answer]
An e-mail and password are not required to play age restricted tracks. Lavaplayer will attempt to play them anyway, however specifying those credentials may offer a higher success rate when it comes to playing such tracks.

Refer to the `application.yml.example` file, as that's where you'll need to specify your credentials.

For region-locked, you can use a proxy or you simply setup nodes in different regions

**Reply by pasindu20** [reply]
can you give me the example i don't now lavalink staff that much

**Reply by devoxin** [reply]
> can you give me the example i don't now lavalink staff that much

It's self-explanatory. Refer to the [example config](https://github.com/freyacodes/Lavalink/blob/dev/LavalinkServer/application.yml.example#L32-L34) where it has comments about what each property is for.
